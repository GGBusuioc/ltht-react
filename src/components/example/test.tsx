import React from 'react'
import { mount, shallow } from 'enzyme'
import renderer from 'react-test-renderer'

import Example from './index'

describe('button', () => {
  describe('basics', () => {
    it('class name', () => {
      const wrapper = mount(<Example>Example</Example>)
      console.log(wrapper.html())
      expect(wrapper.find('h1').hasClass('css-cljjpi')).toBe(true)
    })
    it('should render a h1', () => {
      const wrapper = mount(<Example>Example</Example>)
      expect(wrapper.find('h1').exists()).toBe(true)
    })
    it('matches snapshot', () => {
      const wrapper = mount(<Example>Example</Example>)
      expect(wrapper).toMatchSnapshot()
    })
  })
})
