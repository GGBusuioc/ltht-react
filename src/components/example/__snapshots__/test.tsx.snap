// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button basics matches snapshot 1`] = `
.emotion-0 {
  font-family: Arial,sans-serif;
  font-size: 1.5em;
  text-align: center;
  color: palevioletred;
}

<Styled(h1)>
  <h1
    className="emotion-0"
  >
    Example
  </h1>
</Styled(h1)>
`;
